File: Java/22_BitManipulation/LC_389_FindTheDifference.java
import java.util.HashMap;
import java.util.Map;

class Solution {
    public char findTheDifference(String s, String t) {
        int asciiValue = 0;
        for (char c : s.toCharArray()) {
            asciiValue ^= (int) c;
        }
        for (char c : t.toCharArray()) {
            asciiValue ^= (int) c;
        }
        return (char) asciiValue;
    }
}

public class LC_389_FindTheDifference {

}

File: Swift/Leetcode.playground/Sources/22_BitManipulation/LC_389_FindTheDifference.swift
class Solution {
    func findTheDifference(_ s: String, _ t: String) -> Character {
        var asciiValue: UInt8 = 0
        for c in s { asciiValue ^= c.asciiValue! }
        for c in t { asciiValue ^= c.asciiValue! }
        return Character(UnicodeScalar(asciiValue))
    }
}